R/expect_identical_linter.R

```diff
@@ -69,6 +69,10 @@ expect_identical_linter <- function() {
         and expr[NUM_CONST[contains(text(), '.')]]
       ]
       or following-sibling::expr[NUM_CONST[contains(text(), '.')]]
+      or following-sibling::expr[
```

Comment:
IINM this will skip any subtraction with a number on one side, e.g. x-1.01 or 1.01-x